### DISCUSSION TITLE
Support for HLS m3u8 in src

### DISCUSSION THREAD

**Discussion by BorghildSelle** [muxinc/next-video#289] [Q&A] [answered]
Hello,
we want to use Player only for Screen9 urls, but need support for HLS m3u8 files in src then.
Could that be supported in one of the future versions?

**Comment by luwes** [comment, marked as answer]
Thanks for the question. m3u8 files are supported, for this you wouldn't need the upload workflow so you can use the player directly. see here for more info: https://github.com/muxinc/next-video#player-only-demo

**Reply by frankoprifti** [reply]
I am using this but i get black screen and I can see only this error on server logs
Check the render method of `HlsVideo`
